submissions/ZAIRIyoussef.py

```diff
@@ -0,0 +1,44 @@
+import csv
+from collections import defaultdict
+
+def traiter_donnees(chemin_fichier):
+    prix_par_ville = defaultdict(float)
+    prix_par_produit_et_ville = defaultdict(lambda: defaultdict(list))
+
+    with open(chemin_fichier, newline='', encoding='utf-8') as csvfile:
```

Comment:
Wouldn't this load the whole file in memory ? If yes, it's likely to trigger an OOM